.github/workflows/verify-docker-image.yml

```diff
@@ -59,8 +59,27 @@ jobs:
       - name: Manage py check
         uses: ./.github/actions/run-docker
         with:
-          image: ${{ steps.build.outputs.tags }}
-          options:
+          version: ${{ steps.build.outputs.version }}
           run: |
-            make update_deps
             make check
+
+  test:
```

Comment:
Consider removing that for now - we've proven that it works, but until we do the full transition (PR in the very near future) we don't need to run these.

Reply:
Done.